Cython/TestUtils.py

```diff
@@ -396,3 +397,27 @@ def write_newer_file(file_path, newer_than, content, dedent=False, encoding=None
 
     while other_time is None or other_time >= os.path.getmtime(file_path):
         write_file(file_path, content, dedent=dedent, encoding=encoding)
+
+
+def py_parse_code(code):
```

Comment:
Maybe `parse_python_code` ?
```suggestion
def parse_python_code(code):
```